src/time.cr

```diff
@@ -510,8 +510,12 @@ struct Time
     when utc?
       io << " UTC"
     when incremental?
-      Format.new(" %:z").format(self, io)
-      io << " " << location.name unless local? || location.fixed?
+      if offset % 60 == 0
+        Format.new(" %:z").format(self, io)
+      else
+        Format.new(" %::z").format(self, io)
+      end
+      io << " " << location.name unless location.fixed?
```

Comment:
`" "` could be a `Char`.

Reply:
Thanks, I intended to change that.
However, I'd rather welcome feedback on the general concept instead of such minor details ;)